src/utils/RequestMiddleware.ts

```diff
@@ -0,0 +1,293 @@
+import * as express from "express";
```

Comment:
we must move this code to `italia-ts-commons` - cc @gunzip

Reply:
Ok, I opened a pivotal story and add a TODO comment [#158454734]

Reply:
moved to italia-ts-commons 2.5.0, please use that